tests/global_reactor.py

```diff
@@ -31,14 +31,19 @@ def start_reactor(cls, models, reactor_hook_callbacks):
         for specs in models:
             switch_descriptor = specs["switch_descriptor"]
 
+            hostname = switch_descriptor.hostname
+            m = re.match('^https?:\/\/(.*)$', switch_descriptor.hostname.lower())
```

Comment:
urlparse fait ce genre de trucs la
urlparse(switch_descriptor).scheme == "https"

Reply:
I have encountered a problem using urlparse when your address looks like '127.0.0.1:80/'
What I could see is that it scans for ':' and gives '127.0.0.1' to the sceme instead of leaving it empty...